assets/locales/en.json

```diff
@@ -228,7 +228,7 @@
     "takeScreenshotShortcut": "Global Screenshot Shortcut",
     "downloadHashShortcut": "Global Download Shortcut",
     "experiments": "Experiments",
-    "npmOnIpfs": "npm on IPFS"
+    "npmOnIpfs": "Install npm on IPFS"
```

Comment:
```suggestion
    "npmOnIpfs": "Enable npm on IPFS"
```

Reply:
I'd specifically asked to use "Enable" since it's a little more accurate, and avoids visual repetition. If we' use "Enable" for each one, we might as well remove the word "Enable" altogether.

Reply:
Let's go with Enable here, and after I merge all other PRs we can clean it up later.